src/App.vue

```diff
@@ -2,7 +2,7 @@
   <div>
     <div id="bar-container">
       <CookieBar />
-      <MaintenanceBar />
```

Comment:
why did we remove this?

Reply:
this was just merged into a feature branch and needs several fixes before we merge it to develop